Framework/src/Quality.cxx

```diff
@@ -7,9 +7,10 @@
 
 ClassImp(o2::quality_control::core::Quality)
 
+  // clang-format off
```

Comment:
I guess we should modify clang-format to accept that

Reply:
If you mean the clang-format off/on commands - it accepts them already. 
If you mean moving this line two spaces to the right, this seems to be a bug connected somehow to the usage of the macro above. Do you know to deal with it properly?

Reply:
I had tested the new namespace style with clang-format and it worked. Is it actually not working here ?

Reply:
for me it was suggesting to indent it with two spaces

Reply:
ok, understood